File: Python Contest/area.js

x1 = 2
y1 = 2
x2 = 4
y2 = 4

r = Math.pow((y2-y1),2) + Math.pow((x2-x1),2)

a = 3.14159 * r * r

a = a*1000
a = Math.round(a)
a = a/1000
console.log(a)

File: Python Contest/area.py
import math

def areaOfCircle():
    x1 = input()
    x1 = float(x1)

    y1 = input()
    y1 = float(y1)

    x2 = input()
    x2 = float(x2)

    y2 = input()
    y2 = float(y2)


    r = math.sqrt((x2-x1)*(x2-x1) + (y2-y1)*(y2-y1))

    a = 3.14159*r*r

    a = round(a,3)
    print(a)

areaOfCircle()